cmd/clusterawsadm/ami/helper.go

```diff
@@ -216,7 +216,11 @@ func getAllImages(ec2Client ec2iface.EC2API, ownerID string) (map[string][]*ec2.
 	imagesMap := make(map[string][]*ec2.Image)
 	for _, image := range out.Images {
 		arr := strings.Split(aws.StringValue(image.Name), "-")
-		arr = arr[:len(arr)-2]
+		if arr[len(arr)-2] == "00" {
```

Comment:
Guessing the other way to split is on the length of arr, i.e 6 for old and 5 for new

Reply:
(can't remember if i counted the number correctly though 🤣 )

Reply:
sounds good to me 🙂

Reply:
Hmmm, that's interesting.  You go me thinking, could OS contain additional `-`?

Reply:
Some one could change this:
https://github.com/kubernetes-sigs/image-builder/blob/563c3ae65766a5b5dda910f96015da71263e3a8c/images/capi/packer/ami/amazon-2.json#L4

to:

`"build_name": "amazon-2-ohno"`

Reply:
yeah even I overlooked it. Thanks for rethinking.

Reply:
Thanks @Ankitasw